app/app.go

```diff
@@ -780,8 +741,12 @@ func NewWasmApp(
 			govtypes.ModuleName: gov.NewAppModuleBasic(
 				[]govclient.ProposalHandler{
 					paramsclient.ProposalHandler,
+					// non sdk handlers
+					ibcclientclient.UpdateClientProposalHandler,
+					ibcclientclient.UpgradeProposalHandler,
 				},
 			),
+			ibctm.ModuleName: ibctm.AppModuleBasic{},
```

Comment:
i did have some problem with the unrecognize url type related to ibc light client tendermint types. Dont know if you solve it with this line but i have to add an extra line to register the interface like this `	ibctm.RegisterInterfaces(interfaceRegistry)
` below

Reply:
@sontrinh16 I had the same problem but this line fixes the issue.

But this only brings me to the next problem which is `failed consensus state verification for client (07-tendermint-0): chained membership proof failed to verify membership of value:` . If you have any solution to this, that would be super helpful!